subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -76,15 +77,15 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 struct observe_node {
 	sys_snode_t node;
-	struct lwm2m_obj_path path;
+	sys_slist_t path_list;		/* List of Observation path */
+	bool composite : 1;		/* Composite Observation */
```

Comment:
nit: Could move this at the end of the structure to avoid gaps.

Reply:
ok I will do that.